web/assignments.go

```diff
@@ -140,18 +141,77 @@ func (s *AutograderService) createReview(query *pb.Review) (*pb.Review, error) {
 			submission.ID, assignment.Name, assignment.Reviewers)
 	}
 	query.Edited = time.Now().Format("02 Jan 15:04")
+
```

Comment:
Can this `Format` call use the `layout` const rather than a string as in this case?